bindings/python/CHANGELOG.md

```diff
@@ -61,6 +65,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Switched order of `AddressAndAmount` init params;
 - Renamed `PreparedTransactionData` to `PreparedTransaction`;
 - `{Client, SecretManager}::sign_transaction` return type from `SignedTransactionData` to `TransactionPayload`;
+- Split `Output` into multiple classes;
+- Renamed `TokenScheme` to `SimpleTokenScheme`;
```

Comment:
Ah sorry I didn't see this previously, probably the TokenScheme is added instead of SimpleTokenScheme. Do as you prefer.

Reply:
Used `- `TokenScheme`;` in the added section now